dotnet/SimpleSauce.Test/SauceOptionsTests.cs

```diff
@@ -9,5 +11,56 @@ namespace SimpleSauce.Test
     [TestClass]
     public class SauceOptionsTests : BaseTest
     {
+        [TestInitialize]
+        public void Setup()
+        {
+            SauceOptions = new SauceOptions();
+        }
+
+        [TestMethod]
+        public void UsesLatestChromeWindowsVersionsByDefault()
+        {
+            SauceOptions.BrowserName.Should().BeEquivalentTo(Browser.Chrome);
+            SauceOptions.BrowserVersion.Should().BeEquivalentTo("latest");
+            SauceOptions.PlatformName.Should().BeEquivalentTo(Platforms.Windows10);
+        }
+        [TestMethod]
+        public void UpdatesBrowserBrowserVersionPlatformVersion()
+        {
+            SauceOptions.BrowserVersion = "68";
+            SauceOptions.BrowserName = Browser.Firefox;
+            SauceOptions.PlatformName = Platforms.MacOsHighSierra;
+
+            SauceOptions.BrowserName.Should().BeEquivalentTo(Browser.Firefox);
+            SauceOptions.BrowserVersion.Should().BeEquivalentTo("68");
+            SauceOptions.PlatformName.Should().BeEquivalentTo(Platforms.MacOsHighSierra);
+        }
+        [TestMethod]
+        public void AcceptsAllW3CValues()
+        {
+            SauceOptions.PageLoadStrategy = PageLoadStrategy.Eager;
+            SauceOptions.AcceptInsecureCerts = true;
+            SauceOptions.SetWindowRect = true;
+            SauceOptions.Timeout.Implicit = 4;
+            SauceOptions.Timeout.PageLoad = 44;
+            SauceOptions.Timeout.Script = 33;
+
+            var proxy = new Proxy();
+            SauceOptions.Proxy = proxy;
+            SauceOptions.StrictFileInteractability = true;
+            SauceOptions.UnhandledPromptBehavior = UnhandledPromptBehavior.Dismiss;
+
+            var expectedTimeouts = new Dictionary<string, int> 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'expectedTimeouts'.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4818525)